contrib/lkt/language/parser.py

```diff
@@ -2582,19 +2590,34 @@ def check_correctness_pre():
                     )
                 )
             )
-            # In case of PropertyError, check for RaiseExpr
+            # In case of PropertyError, check presence of RaiseExpr
+            # and correct call to PropertyError exception.
             ._or(
                 If(
-                    And(
-                        rd.result_ref == Entity.property_error_type,
-                        Not(Entity.first_no_paren_parent.is_a(T.RaiseExpr))
-                    ),
+                    Entity.expr_context_free_type
+                    == Entity.property_error_type,
 
-                    Self.error(
-                        S("cannot call ")
-                        .concat(Entity.property_error_type.full_name)
-                        .concat(S(" outside of a raise expression"))
-                    ).singleton,
+                    Cond(
+                        Not(Entity.first_no_paren_parent.is_a(T.RaiseExpr)),
```

Comment:
As we don’t have keyword to make it clear what is a condition and what is an expression for the result, we customarily group corresponding conditions/results together. In a conventional language, where we would have:

```
if C1:
   return E1
elif C2:
   return E2
else:
   return E3
```

we do this in the DSL:

```
Cond(
    C1,
    E1,

    C2,
    E2,

    E3,
)
```

Can you adjust the empty lines accordingly?